src/main/java/longtimenosee/model/event/Date.java

```diff
@@ -18,6 +18,7 @@ public class Date {
     public static final String STANDARD_DATE = "yyyy-MM-dd";
 
     public static final String MESSAGE_FORMAT_CONSTRAINTS = "Dates must follow Format: " + STANDARD_DATE;
+    public static final String RANGE_FORMAT_CONSTRAINTS = "Ensure that Year/Month/Date values entered are appropriate.";
```

Comment:
Maybe we could make this error message clearer.

Perhaps something like:
"Ensure that Year/Month/Date values are valid dates."

Reply:
truee